practice/bio/site.css

```diff
@@ -0,0 +1,41 @@
+body {
+  color: magenta;
+}
+
+div:after {
+  background-image: url("johncena.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  opacity: 0.25;
+}
```

Comment:
This ruleset does not match anything on the page and does not change the background.